CONTRIBUTING.md

```diff
@@ -74,40 +74,57 @@ npm run test -- --watch
 
 The [sandbox](examples/sandbox/README.md) app can be used as a local development aid.
 
+### Branching Strategy
+
+Our branching strategy is based on [Github Flow](https://guides.github.com/introduction/flow/).
```

Comment:
To be honest, I'm not super sure how helpful this link is? It seems to describe GitHub 101 more than a git branching strategy like gitflow or oneflow for example. I think we could probably get away with just removing this paragraph altogether and starting with "Our master branch..." instead.

Reply:
I agree.